src/membraneWebRTC.ts

```diff
@@ -36,8 +36,8 @@ export type BandwidthLimit = number;
 /**
  * Config passed to {@link MembraneWebRTC}.
  */
-export interface MembraneWebRTCConfig {
-  callbacks: Callbacks;
+export interface MembraneWebRTCConfig<TrackMetadata> {
+  callbacks: Callbacks<TrackMetadata>;
 }
```

Comment:
This looks suspiciously. It makes me feel like I did something wrong as I don't feel like Config or Callbacks should be generic but maybe this is a proper way of doing it